vademecum.tex

```diff
@@ -325,6 +325,17 @@ \section{Masters in HPC and SC}
 \end{itemize}
 Starting from from the academic year 2020/2021, SISSA has introduced an important novelty regarding the admission to the MHPC: SISSA waves the fee of the Master to its PhD students who are enrolled on one of its PhD courses on the date of the official start of MHPC classes, and to former students who have obtained their doctorate no more than 12 months before the start date of the lessons and that are currently unemployed, for a maximum of \underline{six} exemptions. The exemption is granted to the first three PhD students and to the first three former PhD students that are admitted to the Master, according to the ranking of the admission. Should the number of admitted students or former students be less than three, the exemptions not enjoyed by one of the two categories will be transferred to the other. Beneficiaries of the exemption are required to complete the training course, under penalty of paying € 1.000,00 to SISSA, as a contribution to the training costs. Beware that, if the beneficiary of an exemption is awarded a scholarship that also includes a contribution for the fees (for example, a contribution offered by other research institutions or private companies), the benefit of the exemption is withdrawn and the student is required to pay the balance of the entire fee for participation in MHPC. If this disclaimer has ignited your interest, have a look at how the application process works at \href{https://www.mhpc.it/how-apply}{this link}!
 
+\section{Temporary unemployment benefit}
+
+If you do not have a Gestiona Separata completed -- it is the first step to do.
+\begin{itemize}
+   \item The first way is to go directly to INPS office Trieste ( Via Sant'Anastasio, 5,), nearby busstop 38. Probably I didn't even fix an appointment- just came and had a talk. In principle, they should give you INPS code. But I had some problems, so I have obtained a SPID code (see (2.) )
+   \item One can obtain a SPID code online having the Tessera Sanitaria (your health card). If it is expired, it's also fine, the main thing is its' number. Although you should fill in the date of expiration of the health card of this year (like, September 2022 or December 2022), even it has already expired -- they do not check it further, the thing which is needed is a number.
+   \item Once you got your SPID, you go to the INPS website (https://www.inps.it/) and complete the procedure "Gestiona Separata".
+   \item Then, you go to Anpal website (https://www.anpal.gov.it/did) and complete the procedure "Did online"-- at the end they will ask you to send an email to schedule an appointment to the adress (cpi.trieste@regione.fvg.it). They will reply to you with a date of the appointment . Then you will go to the address (Scala dei Cappuccini, 1) and get a bunch of documents which one can also use to get a job seeking permit of stay.
```

Comment:
Then, you should go to the ANPAL website (https://www.anpal.gov.it/did) and access to the procedure "Did online". In the end, they will ask you to send an e-mail to schedule an appointment at the address cpi.trieste@regione.fvg.it. They will reply to you to fix an official appointment date. Then, you are required to go to the indicated address (Scala dei Cappuccini, 1) and get a bunch of documents, which one can also use to get a job seeking permit of stay.